parsing/dialects/dialect1/config.fmpp

```diff
@@ -85,6 +87,9 @@ data: {
       "org.apache.calcite.sql.SqlPeriodTypeNameSpec",
       "org.apache.calcite.sql.SqlPeriodTypeNameSpec.TimeScale",
       "org.apache.calcite.sql.SqlPrimaryIndex",
+      "org.apache.calcite.sql.SqlRangeN",
+      "org.apache.calcite.sql.SqlRangeNStartEnd",
+      "org.apache.calcite.sql.SqlRangeN.NoRangeUnknown",
```

Comment:
nit: Alphabetically, I think SqlRangeN.NoRangeUnknown should come before SqlRangeNStartEnd?

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/212/commits/2ac5c3010b51f82c19624c1a7aadff8015134548